docs/src/customization/theming.md

```
@@ -4,7 +4,94 @@ description: Customizing the look and feel of Citizen
+::: info DEPRECATION
+Primary colors can be customised further with HSL fallback variables. However, it has been soft-deprecated, and will be discontinued in the future.
+| Property                     | Description	| 
+| ---------------------------- | ----------- | 
+| `--color-progressive-hsl__h` | Hue of primary color, this will affect other text and surface colors |
+| `--color-progressive-hsl__s` | Saturation of primary color |
+| `--color-progressive-hsl__l` | Lightness of primary color |
 :::
+
+Primary colors can also be changed as a whole via `--color-progressive` itself.
+
+## Surface color
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)